src/doc/manifest.md

```diff
@@ -527,9 +527,13 @@ name = "..."
 crate-type = ["dylib"] # could be `staticlib` as well
 ```
 
-The available options are `dylib`, `rlib`, and `staticlib`. You should only use
-this option in a project. Cargo will always compile packages (dependencies)
-based on the requirements of the project that includes them.
+The available options are `dylib`, `rlib`, `staticlib`, and, as of Rust 1.11, 
+`cdylib`. You should only use this option in a project. Cargo will always 
```

Comment:
Oh, I see, this is a re-flow of the previous text. I won't worry about my beef with these sentences for this PR, then :wink: